viewing_party/party.py

```diff
@@ -0,0 +1,142 @@
+def create_movie(movie_title, genre, rating):
+
+    if not movie_title or not genre or not rating:
+        return None
+
+    dict_of_movies = {
+        "title" : movie_title,
+        "genre" : genre,
+        "rating" : rating
+    }
+    return dict_of_movies
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, movie_title):
+    for movie in user_data["watchlist"]:
+        if movie_title in movie["title"]:
+            user_data["watchlist"].remove(movie)
+            add_to_watched(user_data, movie)
+            return user_data
+    return user_data
+
+def get_watched_avg_rating(user_data):
+    
+    if len(user_data["watched"]) == 0:
+        return 0.0
+    else:
+        total = 0
+        for movie in user_data["watched"]:
+            total += movie["rating"]
+    return total / len(user_data["watched"])
+
+def get_most_watched_genre(user_data):
+    
+    genres = {}
+    most_watched = None
+    genre_count = 0
+
+    for movie in user_data["watched"]:
+        if movie["genre"] in genres:
+            genres[movie["genre"]] += 1
+        else:
+            genres[movie["genre"]] = 1
+        
+    for genre in genres.items():
+        if genre[1] > genre_count:
+            genre_count = genre[1]
+            most_watched = genre[0]
```

Comment:
Nice job finding the max!

Syntax-wise, while `.items()` _does_ technically yield a list of pairs (tuples), we can have python assign the parts to named variables which might be a little easier to read. Consider:
```suggestion
    for genre, count in genres.items():
        if count > genre_count:
            genre_count = count
            most_watched = genre
```